# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/tile_cache.rs
# Commit: fc2b3d6448bc
# Full Hash: fc2b3d6448bcc3575fffb1abc923e7e703aeef54
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 8c70bc92ad36 -r fc2b3d6448bc gfx/wr/webrender/src/tile_cache.rs
--- a/gfx/wr/webrender/src/tile_cache.rs	Wed Feb 23 20:49:26 2022 +0000
+++ b/gfx/wr/webrender/src/tile_cache.rs	Wed Feb 23 20:49:27 2022 +0000
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use api::{ColorF, PrimitiveFlags, QualitySettings};
+use api::{ColorF, PrimitiveFlags, QualitySettings, RasterSpace};
 use api::units::*;
 use crate::clip::{ClipChainId, ClipNodeKind, ClipStore, ClipInstance};
 use crate::frame_builder::FrameBuilderConfig;
@@ -629,6 +629,7 @@
         PrimitiveFlags::IS_BACKFACE_VISIBLE,
         prim_list,
         scroll_root,
+        RasterSpace::Screen,
     ));
 
     PictureIndex(pic_index)